src/scalarstats.jl

```diff
@@ -37,6 +37,27 @@ function harmmean(a::RealArray)
     return n / s
 end
 
+# Generalized mean
+"""
+    genmean(a, p)
+
+Return the generalized/power mean with exponent `p` of a real-valued array.
+i.e. returns ((a_1^p+a_2^p+...+a_n^p)/n)^(1/p), where n is the length of the array a.
```

Comment:
"returns" is redundant here (and there's a discrepancy with "return" anyway). Also add double bacticks around the equation, and single backticks around `n` and `a`. You could write the last part simply as "with `n=length(a)`".